internal/pkg/ccloudv2/metrics.go

```diff
@@ -22,35 +19,30 @@ type responseDataPoint struct {
 	Value     float32   `json:"value"`
 }
 
-func newMetricsClient(baseUrl, userAgent string, unsafeTrace, isTest bool) *metricsv2.APIClient {
+type MetricsClient struct {
+	*metricsv2.APIClient
+	authToken string
+}
+
+func NewMetricsClient(url, userAgent string, unsafeTrace bool, authToken string) *MetricsClient {
```

Comment:
```suggestion
func NewMetricsClient(url, userAgent, authToken string, unsafeTrace bool) *MetricsClient {
```

Reply:
I'm trying to match the order that other similar functions use!